.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+**Bug Report**
+
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+
+How are you building/running PDC?
+
+```bash
+...
+```
+
+What is the running configuration file you use?
```

Comment:
currently we don't need a configuration file, maybe ask for environment variables before we switch to config file

Reply:
Changed to running setup